ligo/haskell/src/Ligo/BaseDAO/Types.hs

```diff
@@ -189,17 +178,20 @@ data ConfigL = ConfigL
 
   , cMaxVotingPeriod :: Natural
   , cMinVotingPeriod :: Natural
+
+  , cCustomEntrypoints :: CustomEntrypointsL
   }
   deriving stock (Show)
 
-mkConfigL :: ConfigL
-mkConfigL = ConfigL
+mkConfigL :: [(MText, ByteString)] -> ConfigL
```

Comment:
I think it would be best to use a type synonym here or a named parameter like in `mkStorageL`.